uoft-tapp/tapp-cp
# create scheduled email for offers accepted alert

Port of @michellemtchai's CP PR #32: https://github.com/uoft-tapp/cp/pull/32

- added `HR_ADMIN_EMAIL` and `HR_ADMIN_NAME` to `.env`
- create email template for status email update
- fixed mangled route typo in `routes.rb`
- set email to send under the following conditions:
  - it is a weekday
  - the system has received at least one offer acceptance within the last 24hr (Tuesday - Friday) or the last 3 days (Monday)


To test this the following commands in order:
- `cp dev.env.default .env` 
- `docker-compose run rails-app rake db:migrate`
- **Task Command:** `docker-compose run rails-app rake email:status`
  - docker doesn't have to be running for the task to be used, but the .env and migration has to be set during the first time
- go to the default Gmail Pinned in Slack to check for emails
